linchpin/cli/__init__.py

```diff
@@ -262,6 +299,17 @@ def _write_distilled_context(self, run_data):
         with open(context_file, 'w+') as f:
             f.write(json.dumps(dist_data))
 
+    def _write_latest_run(self):
+
+        latest_run_data = self._get_run_data_by_txid()
+        resources_path = self.get_evar('resources_folder')
+        context_path = '{0}/{1}'.format(self.workspace, resources_path)
+        if not os.path.exists(context_path):
+            os.makedirs(context_path)
+        context_file = '{0}/{1}'.format(context_path, 'linchpin.latest')
+        with open(context_file, 'w+') as f:
+            f.write(json.dumps(latest_run_data))
+
```

Comment:
This is great! I'm excited to see this out there. Additionally, it might be advantageous to turn this into the outputs (to STDERR) at the end of a linchpin run. The [_handle_results function](https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/shell/__init__.py#L24) could be replaced.